### PR TITLE
Popup menu

### PR DESCRIPTION
**Pull request by mrsamsharma**
- Add my function to dynamically load HTML
- Use loops to go through the data and use project specific data for the popup modal
- Used arrays and object to store the project details

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -68,12 +68,46 @@ <h1>I’m Patrick Ndimukulaga Glad to see you!</h1>
...
+                <a class="modal-card-button" href="https://samwfelice.github.io/Portfolio/"
+                target="_blank">See Live <img src="./images/github-icon.svg"></a>
+                <a class="modal-card-button" href="https://github.com/samWfelice/Portfolio" target="_blank">See Source <img
+                  src="./images/source-icon.svg"></a>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div> -->
+    <!--1st porfolio-->
```
Comment (OmarRamoun) [inline index.html]: please remove the unused commented code in order to keep your file clean and professional

**Review by OmarRamoun — CHANGES_REQUESTED** [review]
# 🟡 Almost There! 🏃

## Hi @Kirabo19  👋

Good job so far! 👏. Your Coding Style is Impressive! 💪.

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃

## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** 🌟 that can improve your code quality 💯 and help you **get a better job** 💼.

_I will guide 📔 you through fixing 🔧 these bugs 🐛, just **follow my comments down below** 🔽 and make sure you **don't miss any** of them, in order to get your project approved ✅_

## Highlights 🌟

- ✔️ descriptive commit messages
- ✔️ descriptive PR title and description
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️

_Check the comments under the review._

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@mrramoun](https://github.com/mrramoun)
- This is My **Twitter**: [@ramoun16](https://twitter.com/ramoun16)
- This is My **Slack**: [Ramoun](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Kirabo19) [reply index.html]: Hi MrRamoun, i have fixed the required changes

**Review by OmarRamoun — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi @Kirabo19 👋

Your project is complete! ✔️ 🏁

Finally, **you did it**! 🎉 👏. Your code was very professional 💯 and I can see you have a great chance to be a Great Software Engineer ⛑️.

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉. And Welcome to Microverse 👋.

> **Note:** You can now take a break even if short before you move on to the next project. Happy 😄 Coding 🖥️! and Have a nice day 🌞.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️
- This is My **GitHub**: [@mrramoun](https://github.com/mrramoun)
- This is My **Twitter**: [@ramoun16](https://twitter.com/ramoun16)
- This is My **Slack**: [Ramoun](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**